frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -73,20 +96,51 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
               breadcrumb={
                 <Breadcrumb>
                   {breadcrumbPath}
-                  <BreadcrumbItem isActive>{pipeline?.name || 'Loading...'}</BreadcrumbItem>
+                  <BreadcrumbItem>{pipeline?.name || 'Loading...'}</BreadcrumbItem>
+                  <BreadcrumbItem isActive>{pipelineVersion?.name || 'Loading...'}</BreadcrumbItem>
                 </Breadcrumb>
               }
-              title={pipeline?.name || 'Loading...'}
-              description={
-                pipeline ? <MarkdownView conciseDisplay markdown={pipeline.description} /> : ''
-              }
+              title={pipelineVersion?.name || 'Loading...'}
+              {...(pipeline && {
+                description: (
+                  <MarkdownView conciseDisplay markdown={pipelineVersion?.description} />
+                ),
+              })}
               empty={false}
-              loaded={pipelineLoad && pipelineTemplateLoaded}
+              loaded={isLoaded}
               loadError={templateLoadError}
               headerAction={
-                pipelineLoad &&
-                pipelineTemplateLoaded && (
-                  <PipelineDetailsActions onDelete={() => setDeleting(true)} pipeline={pipeline} />
+                isPipelineVersionLoaded && (
+                  <Flex
+                    spaceItems={{ default: 'spaceItemsMd' }}
+                    alignItems={{ default: 'alignItemsFlexStart' }}
+                  >
+                    <FlexItem>
+                      <PipelineSelector
```

Comment:
It should be set a `maxWidth` here, or the layout will be broken when the name is very long

https://github.com/opendatahub-io/odh-dashboard/assets/37624318/4652a593-ce7d-4624-83aa-435edf1da62f